index.scss

```diff
@@ -1,2 +1,2 @@
 @import 'nebenan-ui-kit/_index.scss';
-@import 'src/**/*.scss';
+@import 'src/**/index.scss';
```

Comment:
nebenan-frontend only imports index.scss files
All other scss files in nebenan-form are called index.scss.